### ISSUE TITLE
Vignette

### ISSUE THREAD

**Issue by cmaimone** [cproctor/qualitative-coding#67]
I think I'm on the most current version?

```
% qc version
qualitative-coding 1.5.5
```

On importing the vignette file:

```
% qc init --import ../demo.qdpx
Traceback (most recent call last):
  File "/Users/christina/.local/bin/qc", line 8, in <module>
    sys.exit(cli())
             ^^^^^
  File "/Users/christina/.local/pipx/venvs/qualitative-coding/lib/python3.11/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/christina/.local/pipx/venvs/qualitative-coding/lib/python3.11/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/Users/christina/.local/pipx/venvs/qualitative-coding/lib/python3.11/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/christina/.local/pipx/venvs/qualitative-coding/lib/python3.11/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/christina/.local/pipx/venvs/qualitative-coding/lib/python3.11/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/christina/.local/pipx/venvs/qualitative-coding/lib/python3.11/site-packages/qualitative_coding/cli/decorators.py", line 15, in command
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/Users/christina/.local/pipx/venvs/qualitative-coding/lib/python3.11/site-packages/qualitative_coding/cli/init.py", line 19, in init
    reader.unpack_project(Path.cwd())
  File "/Users/christina/.local/pipx/venvs/qualitative-coding/lib/python3.11/site-packages/qualitative_coding/refi_qda/reader.py", line 31, in unpack_project
    QCCorpus.initialize(accept_defaults=True)
TypeError: QCCorpus.initialize() got an unexpected keyword argument 'accept_defaults'
```

**Comment by cproctor** [comment]
Ah, I'm sorry--a careless mistake. I didn't have the latest version active when I ran the tests before deployment. This was an error introduced while correcting another issue. 

This is fixed in 1.5.6; all tests passing and I have confirmed that the commands in the vignette work as expected.

**Comment by cproctor** [comment]
re: https://github.com/openjournals/joss-reviews/issues/7031
